### PR TITLE
Feature/dev 64/creator routes

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

File: packages/app/src/ui-kit/atoms/ProtectedRoute/ProtectedRoute.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+interface PropsInterface extends ProtectedRouteInterface {}
+
+const ProtectedRoute: FC<PropsInterface> = (props) => {
+  const {defaultRedirect, hasRights, children, ...rest} = props;
+  return (
+    <Route
+      {...rest}
+      render={() => {
+        return !hasRights || (hasRights && hasRights()) ? (
```
Comment (dmitry-yudakov) [inline packages/app/src/ui-kit/atoms/ProtectedRoute/ProtectedRoute.tsx]:
```suggestion
        return !hasRights || hasRights() ? (
```

**Review by dmitry-yudakov — APPROVED** [review]
approved with small suggestion

File: packages/app/src/scenes/widgets/Widgets.tsx [hunk 2]
```diff
@@ -54,6 +54,7 @@ const Widgets: FC<PropsInterface> = (props) => {
 
   const {t} = useTranslation();
   const {pathname} = useLocation();
+  //const history = useHistory();
```
Comment (FIZLIN) [inline packages/app/src/scenes/widgets/Widgets.tsx]: Unnecessary comment?

File: packages/app/src/scenes/widgets/Widgets.tsx [hunk 3]
```diff
@@ -79,6 +80,7 @@ const Widgets: FC<PropsInterface> = (props) => {
                 icon="leave-left"
                 size="medium"
                 onClick={() => {
+                  //history.push(ROUTES.explore);
```
Comment (FIZLIN) [inline packages/app/src/scenes/widgets/Widgets.tsx]: ditto

**Review by FIZLIN — APPROVED** [review]
LGTM

File: packages/app/src/scenes/widgets/Widgets.tsx (hunk 2 above)
Comment (dmitry-yudakov) [reply packages/app/src/scenes/widgets/Widgets.tsx]: It's not unnecessary, it's a reminder :) and it's also 2:0 for Unity vs Odyssey devs 😆 @PAKovalev

**Pull request merged**